scst-store/security.md

```diff
@@ -0,0 +1,78 @@
+# Security for Supply Chain Security Tools - Store
+
+## Application Security:
+
+### TLS Encryption
+
+The Store requires TLS connection. If certificates are not provided the store will not start. The Store supports TLS v1.2 and TLS v1.3. The Store does not support TLS 1.0, so a downgrade attack cannot happen. TLS 1.0 prohibited under Payment Card Industry Data Security Standard (PCI DSS).
+
+##### Cryptographic Algorthms:
+
+Elliptic Curve:
+```
+CurveP521
+CurveP384
+CurveP256
+```
+
+Cipher Suites:
+```
+TLS_AES_128_GCM_SHA256
+TLS_AES_256_GCM_SHA384
+TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
+TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+```
+
+### Access Controls
+
+The Store uses [kube-rbac-proxy](https://github.com/brancz/kube-rbac-proxy) as the only entry point to the Store's API. Authentication and Authorization must be completed successfully via the `kube-rbac-proxy` before the Store's API accessible.
```

Comment:
`before the Store's API accessible` seems incomplete. Maybe something like `before the Store's API is accessible`